packages/glibc-repo/build.sh

```diff
@@ -2,25 +2,45 @@ TERMUX_PKG_HOMEPAGE=https://github.com/termux/glibc-packages
 TERMUX_PKG_DESCRIPTION="A package repository containing glibc-based programs and libraries"
 TERMUX_PKG_LICENSE="Apache-2.0"
 TERMUX_PKG_MAINTAINER="@termux-pacman"
-TERMUX_PKG_VERSION=1.0
+TERMUX_PKG_VERSION=1.1
 TERMUX_PKG_AUTO_UPDATE=false
+TERMUX_PKG_DEPENDS="termux-keyring"
 TERMUX_PKG_SKIP_SRC_EXTRACT=true
 TERMUX_PKG_PLATFORM_INDEPENDENT=true
+TERMUX_PKG_CONFFILES="
+	etc/apt/sources.list.d/glibc.sources
+"
 
 termux_step_make_install() {
-	mkdir -p $TERMUX_PREFIX/etc/apt/sources.list.d
-	{
-		echo "# The glibc termux repository, with cloudflare cache"
-		echo "deb https://packages-cf.termux.dev/apt/termux-glibc/ glibc stable"
-		echo "# The glibc termux repository, without cloudflare cache"
-		echo "# deb https://packages.termux.dev/apt/termux-glibc/ glibc stable"
-	} > $TERMUX_PREFIX/etc/apt/sources.list.d/glibc.list
+	mkdir -p "$TERMUX_PREFIX/etc/apt/sources.list.d"
+	{ # write deb822 source entries for the x11 repo
+	printf '%s\n' \
+		"# The termux glibc repository, with cloudflare cache" \
+		"Types: deb" \
+		"URIs: https://packages-cf.termux.dev/apt/termux-glibc/" \
+		"Suites: stable" \
+		"Components: main" \
+		"Signed-By: $TERMUX_PREFIX/etc/apt/trusted.gpg.d/grimler.gpg, $TERMUX_PREFIX/etc/apt/trusted.gpg.d/termux-autobuilds.gpg" \
+		"" \
+		"# The termux glibc repository, without cloudflare cache" \
+		"#Types: deb" \
+		"#URIs: https://packages.termux.dev/apt/termux-glibc/" \
+		"#Suites: stable" \
+		"#Components: main" \
+		"#Signed-By: $TERMUX_PREFIX/etc/apt/trusted.gpg.d/grimler.gpg, $TERMUX_PREFIX/etc/apt/trusted.gpg.d/termux-autobuilds.gpg"
+	} > "$TERMUX_PREFIX/etc/apt/sources.list.d/glibc.sources"
```

Comment:
Are you sure you are required to use `{` and `}` here and in all similar places?

Reply:
They are not required here but I prefer having them for clearly marking this as a single block.